api/workers/cleaner.go

```diff
@@ -22,15 +21,15 @@ import (
 func StartCleaner(_ context.Context, store store.Store) error {
 	envs, err := getEnvs()
 	if err != nil {
-		return fmt.Errorf("failed to get the envs: %w", err)
+		return err
```

Comment:
I believe we could maintain the errors built, what provider a better context on what's happened

Reply:
fixed to return with fmt.